src/renderer/corporate/common/corporate-shareholding.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { Grid, CardHeader, Card, CardContent, withStyles } from '@material-ui/core';
 import { Chart } from 'react-google-charts';
-import { getEntityName, getEntityEquity } from './common-helpers.jsx';
+import { getProfileName, getMemberEquity } from './common-helpers.jsx';
```

Comment:
common helpers should have js extension since its not a react component

Reply:
Fixed on 5ae0084

Reply:
There's jsx output for the icon: https://github.com/SelfKeyFoundation/Identity-Wallet/blob/feat/corporate-edit-member/src/renderer/corporate/common/common-helpers.jsx#L13

But, I can move that out of the helper